security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -76,7 +85,7 @@ The following advisory is available for the OpenShift File Integrity Operator 0.
 [id="openshift-file-integrity-operator-0-1-22-bug-fixes"]
 === Bug fixes
 
-* Previously, a system with a File Integrity Operator installed might interrupt the {product-title} update, due to the  `/etc/kubernetes/aide.reinit` file.  This occurred if the `/etc/kubernetes/aide.reinit` file was present, but later removed prior to the `ostree` validation. With this update, `/etc/kubernetes/aide.reinit` is moved to the `/run` directory so that it does not conflict with the {product-title} update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2033311[*BZ#2033311*])
+* Previously, a system with a File Integrity Operator installed might interrupt the {product-title} update, due to the  `/etc/kubernetes/aide.reinit` file. This occurred if the `/etc/kubernetes/aide.reinit` file was present, but later removed prior to the `ostree` validation. With this update, `/etc/kubernetes/aide.reinit` is moved to the `/run` directory so that it does not conflict with the {product-title} update. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2033311[*BZ#2033311*])
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=2033311 should be removed as it has been released long time ago.